src/data/unionData.ts

```diff
@@ -0,0 +1,49 @@
+interface Section {
+  subtitle?: string;
+  paragraph: string | string[];
+  bulletpoints?: string[];
+}
+
+interface ArticleStructure {
+  mainArticle: Section;
+  subsections: {
+    establishment: Section;
+    goals: Section;
+    development: Section;
+  };
+  addendum: Section;
```

Comment:
If we know that the `mainArticle` and `addendum` have no `subtitle` then we can have 2 separate types for those sections